src/main/resources/templates/pps/bankTransferPenaltyReference.html

```diff
@@ -0,0 +1,77 @@
+        <h2 class="govuk-heading-m">What does the penalty reference start with?</h2>
+        <p class="govuk-hint govuk-!-margin-top-3">For example, the penalty reference A1234567 starts with A.</p>
+        <div class="govuk-form-group" th:classappend="${#fields.hasErrors('selectedPenaltyReference')} ? 'govuk-form-group--error' : ''">
+          <fieldset class="govuk-fieldset">
+            <span class="govuk-error-message"
+                  id="selectedPaymentMethod-errorId"
+                  th:if="${#fields.hasErrors('selectedPenaltyReference')}"
+                  th:each="e : ${#fields.errors('selectedPenaltyReference')}" th:text="${e}">
+            </span>
+
+            <div class="govuk-radios" th:each="penaltyReference : ${availablePenaltyReference}" style="margin-bottom:1em;">
+              <div class="govuk-radios__item">
+                <input class="govuk-radios__input piwik-event"
+                       th:id="${penaltyReference}"
+                       name="available-payment-method"
+                       type="radio"
+                       th:value="${penaltyReference}"
+                       th:field="*{selectedPenaltyReference}"
+                       th:errorclass="govuk-error-message"
+                       th:attr="data-event-id=${penaltyReference}"/>
+                <label class="govuk-label govuk-radios__label" th:for="${penaltyReference}" th:text="${penaltyReference}">keyvalue</label>
+              </div>
+            </div>
+
+          </fieldset>
+        </div>
+        <div class="govuk-form-group">
+          <details class="govuk-details piwik-event" data-event-id="Details - Help with the penalty reference" data-module="govuk-details" aria-labelledby="penalty-reference">
+            <summary class="govuk-details__summary" id="penalty-reference">
+              <span class="govuk-details__summary-text">
+                Help with the penalty reference
+              </span>
+            </summary>
+            <div class="govuk-details__text">
+              <p>
+                You'll find the penalty reference at the top of your penalty notice.
+              </p>
+              <p> Email <a href="mailto:enquiries@companieshouse.gov.uk">enquiries@companieshouse.gov.uk</a> for help if:
+              <ul class="govuk-list govuk-list--bullet">
+                <li>your penalty reference does not start with any of these letters</li>
+                <li>you have lost your penalty notice - use 'Lost penalty notice' as the email subject, and tell us as much information as you can about the penalty so we can pass your message to the right team</li>
+              </ul>
+              <div id="penalty-notice-image"></div>
+            </div>
+          </details>
+        </div>
+        <div class="form-group">
+          <input id="next-button" class="govuk-button piwik-event" data-event-id="Penalty Reference - Continue" type="submit" role="button" value="Continue"/>
```

Comment:
The bit `data-event-id="Penalty Reference -` probably needs to be `data-event-id="Bank Transfer Penalty Reference -`?

Reply:
Updated from the review suggested